cli/src/commands/pull/pull.test.ts

```diff
@@ -27,6 +27,60 @@ const promptsMock = prompts as unknown as ReturnType<typeof vi.fn>;
 const REGION = 'us-east-1';
 const baseUrl = `https://test-1234.${REGION}.xata.sh/db/db1:main`;
 
+const regularMigration1: Schemas.MigrationObject = {
+  id: 'mig_ce3lg2hp3o0em98s8r50',
+  parentID: 'mig_ce3lfvhp3o0em98s8r40',
+  checksum: '1:92d84ef84afc56e2152fd48d098d1b7ef4328217eadd5db6b3f646ac94a1a5ad',
+  operations: [
+    {
+      addColumn: {
+        column: {
+          name: 'test',
+          type: 'string'
+        },
+        table: 'test'
+      }
+    }
+  ]
+};
+
+const pgrollMigration1: Schemas.PgRollMigrationHistoryItem = {
+  done: false,
+  migration: `{"name": "mig_cmkjcdrj7c92neg7lnmg", "operations": [{"drop_column": {"down": "", "table": "tester", "column": "Firstname"}}]}`,
```

Comment:
If I'm not mistaken these are the mocked files in file storage, right? Shouldn't these be JSON.stringified as if they were a result of a `xata pull`?

Reply:
These are the responses from the endpoint so `migration` field will be a string at this point. I'm reading the response and then parsing this into an object before saving to disk. 
When I read each file individually using this data, I call JSON.parse on the data first because that's the format it will have been written in.

Reply:
Okay cool!